__tests__/pages/webhooks/checkout-calculate-taxes.test.ts

```diff
@@ -130,8 +132,41 @@ describe("api/webhooks/checkout-calculate-taxes", () => {
     expect(data.shipping_tax_rate).toBe("0.23");
     expect(data.lines.length).toBe(1);
     expect(data.lines[0].total_gross_amount).toBe("34.44");
-    expect(data.lines[0].total_net_amount).toBe("28");
+    expect(data.lines[0].total_net_amount).toBe("28.00");
     expect(data.lines[0].tax_rate).toBe("0.23");
     expect(res.statusCode).toBe(200);
   });
+
+  it("propagates discounts over lines", async () => {
+    const mockedTaxJarResponseData = dummyFetchTaxesResponse;
+    const mockedTaxJarResponse =
+      mockedFetchTaxesForCheckout.mockImplementationOnce(() => {
+        return mockedTaxJarResponseData;
+      });
+    const { req, res } = mockRequest({
+      method: "POST",
+      event: "checkout_calculate_taxes",
+      domain: "example.com",
+    });
+
+    const checkoutPayload = dummyCheckoutPayload;
+    checkoutPayload.discounts = [{amount: "2"}, {amount: "1"}]
+    const linePayload = checkoutPayload.lines[0]
+    checkoutPayload.lines = [linePayload, linePayload]
+    req.body = [checkoutPayload];
+
+    // @ts-ignore
```

Comment:
Why is `ts-ignore` used here? Plz add comment 😅 

If the TS error cannot be fixed [`@ts-expect-error`](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#-ts-expect-error-comments) would be better here because if it somehow starts working again TS will warn you and you can remove that comment ;)